src/build.gradle

```diff
@@ -28,6 +29,18 @@ subprojects {
     apply plugin: 'org.jetbrains.kotlin.jvm'
     apply plugin: 'org.springframework.boot'
     apply plugin: "kotlin-spring"
+    apply plugin: "io.gitlab.arturbosch.detekt"
+
+    detekt {
```

Comment:
Just highlighting that this is where the task has been added and configured, amongst all the millions of re-formatted files in this PR!